diff --git a/drivers/net/wireless/ath/ath11k/mac.c b/drivers/net/wireless/ath/ath11k/mac.c
index 8a7f2d5..d2f9c79 100644
--- a/drivers/net/wireless/ath/ath11k/mac.c
+++ b/drivers/net/wireless/ath/ath11k/mac.c
@@ -4515,11 +4515,13 @@ static void ath11k_mac_sta_assoc_complete(struct ath11k_vif *vif,
 
 	ath11k_mac_sta_set_supported_rates(ar, sta);
 
-	/* Restrict NSS for MediaTek STA on 160MHz HE mode */
-	if (vif->txrate_he &&
-	    chandef->width == NL80211_CHAN_WIDTH_160 &&
-	    ath11k_sta_is_mediatek(sta))
-		sta->nss_max_rx = 1;
+/*
+	/* Restrict NSS for MediaTek STA on 160MHz HE mode */
+	if (vif->txrate_he &&
+	    chandef->width == NL80211_CHAN_WIDTH_160 &&
+	    ath11k_sta_is_mediatek(sta))
+		sta->nss_max_rx = 1;
+*/
 
 	ath11k_mac_sta_apply_twt_cfg(vif, sta);
 }
